app/src/main/res/layout/table_recylcer_item.xml

```diff
@@ -1,29 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="vertical"
-    >
+    android:orientation="vertical">
 
     <TextView
+        android:id="@+id/column"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/column"
         android:padding="5dp"
-        android:textStyle="bold"
-        />
+        android:textColor="@color/md_black_1000"
+        android:textStyle="bold" />
+
     <TextView
+        android:id="@+id/data_item"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:padding="@dimen/message_layout_margin"
-        android:visibility="gone"
-        android:id="@+id/data_item"
-        />
+        android:visibility="gone" />
+
     <TextView
+        android:id="@+id/data_item_link"
```

Comment:
Camel Case

Reply:
it is used with java file so not applicable